core/src/main/java/org/opensearch/sql/calcite/utils/BuiltinFunctionUtils.java

```diff
@@ -243,6 +243,9 @@ static List<RexNode> translateArgument(
                     context.rexBuilder.makeLiteral(" ")));
         RTrimArgs.addAll(argList);
         return RTrimArgs;
+      case "STRCMP":
+        List<RexNode> StrcmpArgs = List.of(argList.get(1), argList.get(0));
```

Comment:
just wonder why some function's arguments are swapped in this method?

Reply:
It's because the order in `compareTo` between V2 and Calcite is different:
V2:
```
Integer.compare(str1.stringValue().compareTo(str2.stringValue()), 0)))
```
Calcite
```
return (int) Math.signum(s1.compareTo(s0));
```